TAO/tao/Cache_Entries_T.inl

```diff
@@ -128,6 +126,9 @@ namespace TAO
   {
     if (this != &rhs)
       {
+        if (this->is_delete_)
+          delete this->transport_property_;
```

Comment:
The very next line in the same file assigns the value to this->transport_property_. I will add setting to 0 to stay exception neutral